docker.go

```diff
@@ -410,6 +418,42 @@ func (p Plugin) Exec() error {
 		}
 	}
 
+	if p.TarPath != "" && p.Dryrun {
+		if len(p.Build.Tags) > 0 {
+			tag := p.Build.Tags[0]
+			fullImageName := fmt.Sprintf("%s:%s", p.Build.Repo, tag)
+
+			if !p.Build.BuildxLoad {
+				fmt.Printf("Warning: To save images to tar, 'load' must be enabled. Adding the '--load' flag automatically.\n")
+				p.Build.BuildxLoad = true
```

Comment:
Do we need this? Why should we expect the user to give build arg `--load`?

Reply:
this will be removed as `--load` is set already even before this call -| https://github.com/drone-plugins/drone-buildx/blob/6d4435fe34fe14a8ec3e8730e1b3a7a7e06e9309/docker.go#L622-L628